django/db/backends/mysql/introspection.py

```diff
@@ -298,7 +288,7 @@ def get_constraints(self, cursor, table_name):
                 constraints[index] = {
                     'columns': OrderedSet(),
                     'primary_key': False,
-                    'unique': False,
```

Comment:
All changes to MySQL introspection (except this line) can be treated  as a simplification of the current implementation, so I moved it to a separate PR #14000.